packages/manager/modules/sms/src/sms/dashboard/telecom-sms-dashboard.controller.js

```diff
@@ -43,42 +54,71 @@ export default class {
+              hit: 'sms::service::dashboard::shortcuts::add-receivers',
+            },
+          ]
+        : []),
       {
         name: 'manage_senders',
         sref: 'sms.service.senders.add',
         text: this.$translate.instant('sms_actions_create_sender'),
         hit: 'sms::service::dashboard::shortcuts::add-senders',
       },
-      {
-        name: 'manage_soapi_users',
-        sref: 'sms.service.users',
-        text: this.$translate.instant('sms_actions_create_api_user'),
-        hit: 'sms::service::dashboard::shortcuts::add-user',
-      },
-      {
-        name: 'manage_blacklisted_senders',
-        sref: 'sms.service.receivers',
-        text: this.$translate.instant('sms_actions_clean_contact_list'),
-        hit: 'sms::service::dashboard::shortcuts::clean-receivers',
-      },
-      {
-        name: 'create_campaign',
-        sref: 'sms.service.batches.create',
-        text: this.$translate.instant('sms_actions_create_campaign'),
-        hit: 'sms::service::dashboard::shortcuts::add-campaign',
-      },
-      {
-        name: 'campaign_history',
-        sref: 'sms.service.batches.history',
-        text: this.$translate.instant('sms_actions_campaign_history'),
-        hit: 'sms::service::dashboard::shortcuts::historic-campaigns',
-      },
+      ...(!this.isSmppAccount
+        ? [
+            {
+              name: 'manage_soapi_users',
+              sref: 'sms.service.users',
+              text: this.$translate.instant('sms_actions_create_api_user'),
+              hit: 'sms::service::dashboard::shortcuts::add-user',
+            },
+          ]
+        : []),
+      ...(!this.isSmppAccount
+        ? [
+            {
+              name: 'manage_blacklisted_senders',
+              sref: 'sms.service.receivers',
+              text: this.$translate.instant('sms_actions_clean_contact_list'),
+              hit: 'sms::service::dashboard::shortcuts::clean-receivers',
+            },
+          ]
+        : []),
+      ...(!this.isSmppAccount
+        ? [
+            {
+              name: 'create_campaign',
+              sref: 'sms.service.batches.create',
+              text: this.$translate.instant('sms_actions_create_campaign'),
+              hit: 'sms::service::dashboard::shortcuts::add-campaign',
+            },
+          ]
+        : []),
+      ...(!this.isSmppAccount
+        ? [
+            {
+              name: 'campaign_history',
+              sref: 'sms.service.batches.history',
+              text: this.$translate.instant('sms_actions_campaign_history'),
+              hit: 'sms::service::dashboard::shortcuts::historic-campaigns',
+            },
+          ]
+        : []),
+      ...(this.isSmppAccount
+        ? [
+            {
+              name: 'option_smpp_parameter',
+              sref: 'sms.service.options.smppParameter',
+              text: this.$translate.instant('sms_actions_smpp_parameter'),
+            },
```

Comment:
regarding others object, it seems we missed the `hit` param.
Is it expected ?

Reply:
Yes, I have a dedicated topics for the tracking.